irohad/ametsuchi/impl/mutable_storage_impl.cpp

```diff
@@ -49,11 +49,11 @@ namespace iroha {
         const model::Block &block,
         std::function<bool(const model::Block &, WsvQuery &, const hash256_t &)>
             function) {
-      auto execute_command = [this](auto command) {
-        return command_executors_->getCommandExecutor(command)->execute(
-            *command, *wsv_, *executor_);
-      };
-      auto execute_transaction = [execute_command](auto &transaction) {
+      auto execute_transaction = [this](auto &transaction) {
```

Comment:
Maybe just inline?

Reply:
A separate variable is used for readability.